### PR TITLE
refactor: Package exports

### PR DESCRIPTION
**Pull request by 0xShisui**

### CODE REVIEW

File: doc/classes/LooksRare.LooksRare.md [hunk 1]
```diff
@@ -269,14 +293,43 @@ Create a taker ask order for collection order.
...
+ - this.createTaker
+ - this.createMakerCollectionOfferWithMerkleTree
 
 #### Parameters
 
 | Name | Type | Description |
 | :------ | :------ | :------ |
 | `maker` | [`Maker`](../interfaces/types.Maker.md) | - |
 | `itemId` | `BigNumberish` | Token id to use as a counterparty for the collection order |
+| `itemIds` | `BigNumberish`[] | List of item ids used during the maker creation |
```
Comment (hexacroes) [inline doc/classes/LooksRare.LooksRare.md:332]:
```suggestion
| `itemIds` | `BigNumberish`[] | List of token ids used during the maker creation |
```

File: doc/interfaces/types.Maker.md [hunk 2]
```diff
@@ -90,7 +90,7 @@ ___
 
 • **quoteType**: [`QuoteType`](../enums/types.QuoteType.md)
 
-Ask or bid
+Bid or ask, 0: bid, 1: ask
```
Comment (hexacroes) [inline doc/interfaces/types.Maker.md:93]:
```suggestion
Bid or Ask, 0: bid, 1: ask
```

File: src/utils/eip712.ts [hunk 3]
```diff
@@ -0,0 +1,97 @@
+import { utils } from "ethers";
+import { EIP712TypedData, Maker } from "../types";
+
+// EIP 712 (Typed structured data hashing and signing) related data
+// https://eips.ethereum.org/EIPS/eip-712
+
+export const MAKER_HASH = "0x003c1bce41a2de73dfe64d6eeb2b3d7f15f1c0c382d9d963c2c6daeb75f0e539";
```
Comment (hexacroes) [inline src/utils/eip712.ts:7]: As a constant, should that be in `src/constants/eip712.ts`?

**Review by hexacroes — APPROVED** [review]

File: src/utils/eip712.ts (hunk 3 above)
Comment (0xShisui) [reply src/utils/eip712.ts:7]: it's only used in this file tho, I'm even considering removing the export

**Pull request merged**